core/labeled-replies-3.3.md

```diff
@@ -40,7 +40,7 @@ Additionally, labeled responses allow bouncers with multiple connected clients t
 
 ### Dependencies
 
-This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and relies on support for the [`batch`](/specs/extensions/batch-3.2.html) capability which MUST be negotiated at the same time.
+This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and uses the [`batch`](/specs/extensions/batch-3.2.html) capability which SHOULD be negotiated at the same time.
```

Comment:
I tried to find the most recent relevant commit that addressed the issue - still finding my way around GitHub in a collaborative sense. I'll always comment on the latest in the future.

> SHOULD means recommended not required.

I'm being a bit nit-picky here and the language seems to suggest that it is always recommended, rather than recommended only in cases where it may be needed.